release-lease.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: jumpstarter-release-lease
+  namespace: jumpstarter-pipelines
+spec:
+  params:
+    - description: The lease to be released.
+      name: jmp-lease
+      type: string
+    - default: ''
+      description: The client intending to release the lease.
+      name: client-name
+      type: string
+    - default: 'no'
+      description: Sets if the client connection to openshift is secure or not.
+      name: grpc_insecure
+      type: string
```

Comment:
same as for the other task, I would skip this all together and let the customer set
```
tls:
   insecure: True
```

on the config file.